### PR TITLE
Issue 3624 blog v0.23

### LINKED ISSUE #3624: Blog post for Knative release v0.23
Blog post for Knative release v0.23

Add post to https://knative.dev/blog/releases/

### PR DESCRIPTION
**Pull request by csantanapr**
Blog release for v0.23
Fixes #3624

### CODE REVIEW

File: blog/releases/announcing-knative-v0-17-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-17-release.md:2]: According to https://docs.google.com/spreadsheets/d/1p1_kBUd6ZvonxHkMcEJPayf6QIpExuFf5cFq0ptar7I/edit#gid=0 we're using v0.x now instead

File: blog/releases/announcing-knative-v0-18-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-18-release.md:2]: see above and https://docs.google.com/spreadsheets/d/1p1_kBUd6ZvonxHkMcEJPayf6QIpExuFf5cFq0ptar7I/edit#gid=0

File: blog/releases/announcing-knative-v0-19-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-19-release.md:2]: see above and https://docs.google.com/spreadsheets/d/1p1_kBUd6ZvonxHkMcEJPayf6QIpExuFf5cFq0ptar7I/edit#gid=0

File: blog/releases/announcing-knative-v0-20-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-20-release.md:2]: see above and https://docs.google.com/spreadsheets/d/1p1_kBUd6ZvonxHkMcEJPayf6QIpExuFf5cFq0ptar7I/edit#gid=0

File: blog/releases/announcing-knative-v0-21-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-21-release.md:2]: see above and https://docs.google.com/spreadsheets/d/1p1_kBUd6ZvonxHkMcEJPayf6QIpExuFf5cFq0ptar7I/edit#gid=0

File: blog/releases/announcing-knative-v0-23-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-23-release.md:3]: see above and https://docs.google.com/spreadsheets/d/1p1_kBUd6ZvonxHkMcEJPayf6QIpExuFf5cFq0ptar7I/edit#gid=0

File: blog/releases/announcing-knative-v0-23-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-23-release.md:50]:
```suggestion
- The stats scraping in the autoscaler is now sensitive to the `EnableMeshPodAddressability` setting. A restart of the autoscaler is required for the setting to take effect if changed. [#11161](https://github.com/knative/serving/pull/11161)
- The state keeping in the activator is now sensitive to the `EnableMeshPodAddressability` setting. A restart of the activator is required for the setting to take effect if changed. [#11172](https://github.com/knative/serving/pull/11172)
- Tightens the heuristic for mesh being enabled in the service scraper. We now expect all errors to be related to mesh (i.e. 503 status code). This prevents accidentally falling in to service scrape mode when errors are encountered for other reasons. [#11174](https://github.com/knative/serving/pull/11174)
```

File: blog/releases/announcing-knative-v0-23-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-23-release.md:72]: Is there any docs work to do here to make sure any sample YAMLs are showing the right API version?

File: blog/releases/announcing-knative-v0-23-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-23-release.md:89]:
```suggestion
- Sequence now displays conditions as `Unknown` rather than `False` for non-terminal condition states. [#5369](https://github.com/knative/eventing/pull/5369)
```

File: blog/releases/announcing-knative-v0-23-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-23-release.md:90]:
```suggestion
- Running validation for event ingress on multi-tenant broker [#5275](https://github.com/knative/eventing/pull/5275)
```

File: blog/releases/announcing-knative-v0-23-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-23-release.md:159]:
```suggestion
- Simplify Istio installation for end-to-end test. [#564](https://github.com/knative/operator/pull/564)
```

File: blog/releases/announcing-knative-v0-23-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-23-release.md:160]:
```suggestion
- Use the `deployment` namespace to determine gateway namespace for Kourier. [#577](https://github.com/knative/operator/pull/577)
```
Is it actually called `deployment`?

File: blog/releases/announcing-knative-v0-23-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-23-release.md:161]:
```suggestion
- Updated the Knative Serving upgrade tests script. [#574](https://github.com/knative/operator/pull/574)
```

File: blog/releases/announcing-knative-v0-23-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-23-release.md:57]:
```suggestion
- Revision replicas now shut down 15 seconds faster. [#11249](https://github.com/knative/serving/pull/11249)
```

File: blog/releases/announcing-knative-v0-23-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-23-release.md:59]:
```suggestion
- The activator proxy now recognizes the `EnableMeshPodAddressability` setting. [#11162](https://github.com/knative/serving/pull/11162)
- Update the User-Agent used during tag resolution [#10590](https://github.com/knative/serving/pull/10590)
```

File: blog/releases/announcing-knative-v0-23-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-23-release.md:76]:
```suggestion
- InMemoryChannel can now be used independently of Knative Eventing, and can be installed by applying the `in-memory-channel.yaml`. [#5355](https://github.com/knative/eventing/pull/5355)
```

Comment (abrennan89) [reply blog/releases/announcing-knative-v0-23-release.md:76]: Should there be docs for things like this?

File: blog/releases/announcing-knative-v0-23-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-23-release.md:77]:
```suggestion
- Multi-tenant, channel-based brokers now have improved filter performance for Triggers with few or no filters. [#5288](https://github.com/knative/eventing/pull/5355)
```

How many is "few", can we specify? e.g. "up to X filters"?

File: blog/releases/announcing-knative-v0-23-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-23-release.md:82]:
```suggestion
- The Trigger `delivery` spec is now automatically propagated to Subscriptions that are connected to multi-tenant, channel-based Brokers. [#5267](https://github.com/knative/eventing/pull/5267)
```

Comment (abrennan89) [reply blog/releases/announcing-knative-v0-23-release.md:82]: idk if some of these are accurate but I'm adding suggestions to improve the wording if this is what it means

File: blog/releases/announcing-knative-v0-23-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-23-release.md:88]:
```suggestion
- Tools from the [`eventing-contrib` repository](https://github.com/knative/eventing-contrib/tree/main/cmd) have been moved to the `eventing` repository. [#5295](https://github.com/knative/eventing/pull/5295)
```

**Review by abrennan89 — COMMENTED** [review]
some suggestions

File: blog/releases/announcing-knative-v0-23-release.md
Comment (csantanapr) [reply blog/releases/announcing-knative-v0-23-release.md:72]: Don't know @knative/eventing-wg-leads would need to look into that

**Comment by csantanapr** [comment]
@abrennan89 thank you for the great review in so short time ❤️ 
I will address the suggestions now 👍

File: blog/releases/announcing-knative-v0-23-release.md
Comment (csantanapr) [reply blog/releases/announcing-knative-v0-23-release.md:76]: Yes they should write docs for this like this @knative/eventing-wg-leads

File: blog/releases/announcing-knative-v0-23-release.md
Comment (csantanapr) [reply blog/releases/announcing-knative-v0-23-release.md:77]: Don't know, but @n3wscott  might know

File: blog/releases/announcing-knative-v0-23-release.md
Comment (csantanapr) [reply blog/releases/announcing-knative-v0-23-release.md:82]: yes this reads better 👍

Comment (csantanapr) [reply blog/releases/announcing-knative-v0-23-release.md:82]: And your input is helping a lot @abrennan89

File: blog/releases/announcing-knative-v0-23-release.md
Comment (csantanapr) [reply blog/releases/announcing-knative-v0-23-release.md:160]: Don't think is a namespace named `deployment`
Looking at the code in the [PR 577](https://github.com/knative/operator/pull/577/files#diff-b4c139ccd3f76f7257855c955ca2d3bceaf5786788d965f88fbcfc34a2d3585dR46)  is setting an environment variable to the namespace value where the kourier gateway deployment is deploy.

**Comment by csantanapr** [comment]
@abrennan89 PTAL when you have chance and lgtm if you think is good to go live

File: blog/releases/announcing-knative-v0-23-release.md
Comment (abrennan89) [inline blog/releases/announcing-knative-v0-23-release.md:2]: I think the idea was that it's just v0.2x rather than "Version v0.2x" 🙂 sorry. If you don't like this as much as "Version 0.2x" I'd raise it with @omerbensaadon maybe since it was added to the word list that way. I don't care which we use as long as it's consistent.

**Review by abrennan89 — APPROVED** [review]
/lgtm
/hold

Other than the naming for the versions this looks fine to me, I'll add a hold in case you wanna update that first?

**Comment by abrennan89** [comment]
Sorry this wasn't the most thorough review btw, I just did a quick one while I had a chance. In the future I'd like to help with blog posts more if I can get time.

@omerbensaadon can you talk with the TOC about release notes processes maybe also, I feel like we could be doing more in terms of editing / reviewing for issues because sometimes things are listed in the blog post where myself and even Carlos aren't sure what they mean because they're not worded too clearly. Maybe Richard and Samia could help with content here also?

File: blog/releases/announcing-knative-v0-23-release.md
Comment (csantanapr) [reply blog/releases/announcing-knative-v0-23-release.md:2]: No problem, I will make the update.
I will go back and remove the "Version" from all.

Comment (csantanapr) [reply blog/releases/announcing-knative-v0-23-release.md]: @abrennan89 I remove "Version " from all titles and linktitles so all are consistent now

File: blog/releases/announcing-knative-v0-23-release.md
Comment (abrennan89) [reply blog/releases/announcing-knative-v0-23-release.md:72]: Ville checked this and put in a PR

**Review by abrennan89 — APPROVED** [review]
/lgtm

**Pull request merged**
